sos/plugins/console.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from glob import glob
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class Console(Plugin, RedHatPlugin):
+    """Console and keyboard information
+    """
+
+    plugin_name = 'console'
+    profiles = ('system',)
+    packages = ('kbd',)
+
+    def setup(self):
+        self.add_copy_spec("/proc/consoles")
+
+        self.add_cmd_output("fgconsole")
+
+        for tty in glob("/dev/tty[0-8]"):
+            self.add_cmd_output("kbdinfo -C " + tty + " gkbled")
```

Comment:
We could reduce this to a single `add_copy_spec()` call via:

```python
self.add_copy_spec([
    "kbdinfo -C %s gkbled" % tty for tty in glob.glob("/dev/tty[0-8]")
])

which imo also improves readability.

Reply:
Thanks, will update so.